src/data/faq_de.json

```
@@ -376,6 +376,217 @@
+                                "title": "Ich habe mein positives Ergebnis aus anderem Wege erfahren (E-Mail, Telefonat…), aber kein Ergebnis in der App – kann ich trotzdem meine Kontakte warnen?",
+                                "anchor": "warn_without_tan_external_result",
+                                "active": false,
+                                "textblock": [
+                                    "Ja, nutzen Sie ab Release 3.0 die neue Funktion \"Tests erfassen und andere warnen\", dazu ist ggf. ein Update der App nötig.",
+                                    "Warnen Sie damit Andere auf Basis des empfangenen positiven Ergebnisses manuell in der CWA.",
+                                    "Siehe: <a href='placeholder' target='_blank'>Wie kann ich andere Nutzer warnen, wenn ich positiv getestet worden bin?</a>"
+                                ]
+                            },
+                            {
+                                "title": "Muss ich trotzdem die Hotline anrufen?",
+                                "anchor": "warn_without_tan_hotline",
+                                "active": false,
+                                "textblock": [
+                                    "Nein, ab Release 3.0 der Corona-Warn-App ist es nicht mehr notwendig, die Verifikations-Hotline anzurufen."
+                                ]
+                            },
+                            {
+                                "title": "Muss ich auf das neue Release umsteigen, oder läuft meine aktuelle Version der CWA weiter?",
+                                "anchor": "warn_without_tan_version",
+                                "active": false,
+                                "textblock": [
+                                    "Wenn Sie die Corona-Warn-App nicht auf das neue Release aktualisieren, wird für eine Übergangsfrist (bis zum ??????h) noch die Verifikations-Hotline zur Verfügung stehen, bei der Sie sich eine TAN ausstellen lassen können.",
+                                    "Nach diesem Zeitpunkt werden entweder die Testergebnisse nur noch direkt digital von den Teststellen in die CWA übertragen, wie bisher haben Sie dann die Möglichkeit, andere zu warnen. Oder Sie erhalten das Testergebnis auf anderem Wege. Dann können Sie – nach einem Update auf Release 3.0 – manuell mit der neuen Funktion \"Tests erfassen und andere warnen\" andere direkt warnen.",
+                                    "Für alle anderen Funktionen unterscheidet sich das neue Release von der vorherigen Version der CWA nicht.",
+                                    "Siehe auch: <a href='placeholder' target='_blank'>Wie kann ich andere Nutzer warnen, wenn ich positiv getestet worden bin?</a>"
+                                ]
+                            },
+                            {
+                                "title": "Muss ich einen PCR-Test machen, nachdem ich mit einem Selbsttest (positives Ergebnis) gewarnt habe?",

```

Improved & easier wording

```suggestion
                                "title": "Muss ich einen PCR-Test machen, nachdem ich mit einem positiven Selbsttest gewarnt habe?",
```

---

Your suggestion has been comitted, thanks